salesforce/refocus
# if the resource contains invalid keys in hmset, log the resource

to test:
- turn on toggles `ENABLE_REDIS_SAMPLE_STORE,` `LOG_INVALID_HMSET_VALUES`
- run  `mocha tests/db/model/aspect/update.js`
- should get this output
```
Trace: Invalid hmset params found when setting samsto:aspect:___testaspect, received{"name":"___TestAspect","timeout":"1s","isPublished":true,"relatedLinks":"[{\"name\":\"new-reLink\",\"url\":\"https://evenfakerlink.com\"}]","id":"49b74d98-a914-410b-bf05-244f3e1bfc5e","isDeleted":"0","valueType":"BOOLEAN","tags":"\"[]\"","createdAt":"2017-09-27T16:29:29.962Z","updatedAt":"2017-09-27T16:29:29.964Z","statusChangedAt":"2017-09-27T16:29:29.966Z"}
    at logInvalidHmsetValues (/Users/anny.he/workspace/refocus/utils/common.js:28:17)
    at Object.hmSet (/Users/anny.he/workspace/refocus/cache/redisOps.js:46:3)
    at afterUpdate (/Users/anny.he/workspace/refocus/db/model/aspect.js:323:22)
    at /Users/anny.he/workspace/refocus/node_modules/sequelize/lib/hooks.js:142:19
    at tryCatcher (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/util.js:16:23)
    at Object.gotValue (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/reduce.js:155:18)
    at Object.gotAccum (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/reduce.js:144:25)
    at Object.tryCatcher (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromiseCtx (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/promise.js:606:10)
    at Async._drainQueue (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/async.js:138:12)
    at Async._drainQueues (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/Users/anny.he/workspace/refocus/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:570:20)
    at tryOnImmediate (timers.js:550:5)
    at processImmediate [as _immediateCallback] (timers.js:529:5)
```
